src/fortranscanner.l

```diff
@@ -654,8 +657,8 @@ SCOPENAME ({ID}{BS}"::"{BS})*
                                           }
 
                                           yy_push_state(Typedef,yyscanner);
-                                          yyextra->current->protection = yyextra->defaultProtection;
-                                          yyextra->typeProtection = yyextra->defaultProtection;
+                                          yyextra->current->protection = Package; // invalid in Fortran, replaced below
+                                          yyextra->typeProtection = Public;
```

Comment:
On which Fortran standard based rule the protection is set to "Public", e.g. module variables are by default Private

Reply:
Default in Fortran is public, from the [F2003 handbook]((https://doi.org/10.1007/978-1-84628-746-6):
> If the accessibility of a particular module entity is not explicitly specified, the moduleʹs default accessibility applies to that > entity. The moduleʹs default accessibility is
> specified by the accessibility statement without an access-id list; if there is no such
> statement, the default accessibility is PUBLIC

Reply:
From head I thought that I remembered that the default accessibility in Fortran was PRIVATE though when I look in the Fortran 2018 standard  I see that it is PUBLIC>
(Been working lately to much with C++ where the default, for a class, is private.)